src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -224,9 +224,26 @@ public final boolean load(SectionNode sectionNode) {
 			// Set the backup interval, if present & enabled
 			if (!"0".equals(getValue(sectionNode, "backup interval"))) {
 				Timespan backupInterval = getValue(sectionNode, "backup interval", Timespan.class);
-
+				boolean purge = getValue(sectionNode, "backup purge", Boolean.class);
+				final Integer required = getValue(sectionNode, "purge required", Integer.class);
```

Comment:
can't you use primitives here like on previous line?

Reply:
@Efnilite I used Integer instead of int solely on the off chance an error occurs or invalid input from a user. Which allows me to check if it's null in the following checks. That way the purge feature can be disabled. If you think I should still go ahead and change it to int I can.